guides/user/managed-delivery/faq/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: single
+title:  "FAQ"
+sidebar:
+  nav: guides
+redirect_from: /reference/managed-delivery/
+---
+
+{% include toc %}
+
+This is a quick reference for Managed Delivery. 
+
+### Where am I?
+
+You probably arrived here from a tooltip on a declaratively managed resource.
```

Comment:
The plan is to send users to an FAQ page from a tooltip?

Reply:
This is just the file that was at `guides/user/managed-delivery/index.md` moved to `guides/user/managed-delivery/faq/index.md`. @emjburns do you know why it contains tooltip related language?

Reply:
Turns out this doc really is deeplinked to from deck, which needs updating. We'll rework this page in a future PR.